IndexedDB/idbfactory-deleteDatabase-opaque-origin.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>IDBFactory.deleteDatabase() and opaque origins</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+
+function load_iframe(src, sandbox) {
+  return new Promise(resolve => {
+    const iframe = document.createElement('iframe');
+    iframe.onload = () => { resolve(iframe); };
+    if (sandbox)
+      iframe.sandbox = sandbox;
+    iframe.srcdoc = src;
+    iframe.style.display = 'none';
+    document.documentElement.appendChild(iframe);
+  });
+}
+
+function wait_for_message() {
+  return new Promise(resolve => {
+    self.addEventListener('message', function listener(e) {
```

Comment:
Depending on how the page loads, this could be a message from the first test or the second test. Perhaps you could pass in the IFrame element and verify something like `e.source == iframe.contentWindow`? That would separate the first frame from the second.